app/context_test.go

```diff
@@ -29,40 +29,40 @@ func TestFromContext(t *testing.T) {
 	parentCtx = context.WithValue(parentCtx, UserIDKey, expectedUserID)
 	parentCtx = context.WithValue(parentCtx, TenantIDKey, expectedTenantID)
 
-	ctx := FromContext(parentCtx)
+	ctx := NewContext(parentCtx)
```

Comment:
FromContext was a good name, because it uses an existing context. It doesn't create one

Reply:
Agree 👍